WORKSPACE

```diff
@@ -1,5 +1,10 @@
 workspace(name = "tf_serving")
 
+local_repository(
+    name = "org_tensorflow",
```

Comment:
Unfortunately this is not possible. It has come up several time but it creates a circle in the dependency graph.

I guess we need to find a solution for that.